json-c.def

```diff
@@ -0,0 +1,150 @@
+EXPORTS
+    ; JSONC_PRIVATE
+    array_list_add
```

Comment:
These symbols should not be listed here.  These are not marked with JSON_EXPORT, and so would not actually be exported on Windows builds.

Reply:
Yep. Currently they are exported on non Windows builds. I’ll work around this for meson.